src/discussions/discussions-home/DiscussionsHome.jsx

```diff
@@ -78,8 +78,8 @@ export default function DiscussionsHome() {
         inContext,
       }}
     >
-      <main className="container-fluid d-flex flex-column p-0">
-        <div className="d-flex flex-row justify-content-between shadow navbar">
+      <main className="container-fluid d-flex flex-column p-0 h-100 w-100">
+        <div className="d-flex flex-row justify-content-between shadow navbar fixed-top">
```

Comment:
Why do we need the `fixed-top`? Imho, `main` itself should not be scrollable, so the header wouldn't move.

Reply:
Fixed top is a slightly hacky way to get that bar higher in the z-index so that it can show the shadow above the other elements when scrolling.